api/prisma/seed-helpers/jurisdiction-factory.ts

```diff
@@ -17,5 +17,6 @@ export const jurisdictionFactory = (
   enableAccessibilityFeatures: true,
   enableUtilitiesIncluded: true,
   enableGeocodingPreferences: true,
+  enableGeocodingRadiusMethod: true,
```

Comment:
Can we flip this to false too please

Reply:
@mcgarrye Do you have any specific reason for that?
It seem like locally it's not a problem to have it switched on.
It also fails tests (which i can change, but probably they will never appear later 😄 ).
So just let me know it you see any problems with that beeing `true` here.

Reply:
Generally we want the seed data to match the actual environment to the best of our ability. And would be good to have a test around the flag

Reply:
It makes sense! I switched it to `false`, and modified test to use second option (so map layer validation). Is that ok ?